examples/pm2-typescript/nuxt.config.ts

```diff
@@ -0,0 +1,7 @@
+export default {
+  hooks: {
+    listen: () => {
+      process.send('ready')
```

Comment:
![screenshot_9](https://user-images.githubusercontent.com/25272043/52071034-7034de80-2582-11e9-8bf5-2825ac5975e1.png)

It seems that we need to at least check `if (process.send)` to prevent TypeScript error.
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/index.d.ts#L848

Reply:
You can also define the hook without async function
```js
export default {
  listen () {
    if (process.send) {
      process.send('ready')
    }
  }
}
```